template/README.md

```diff
@@ -233,7 +233,7 @@ Please be advised that this is also a custom feature of Webpack.
 
 ### Install React Bootstrap
 
-React Bootstrap is not a dependency for using React however it is a popular alternative for integrating Bootstrap with React apps. It can be integrated with Create React App workflow using following three steps.
+React Bootstrap is not a dependency for using React however it is a popular alternative for integrating Bootstrap with React apps. It can be integrated with Create React App workflow following three steps:
```

Comment:
I think the author meant “using the following three steps”. But I agree it’s a bit clumsy.
Let’s reword to completely avoid passive?

`You don’t have to use React Bootstrap together with React but it is a popular library for integrating Bootstrap with React apps. If you need it, you can integrate it with Create React App by following these steps:`